### PR TITLE
fix(eslint-plugin-template): accessibility-valid-aria reporting non a…

### LINKED ISSUE #271: options for valid aria attributes rule
It would be nice if I could provide an exceptions list/ignored properties list for `@angular-eslint/template/accessibility-valid-aria` rule.

I enabled this rule and ended up with this error:
image

In this particular case `variant` is an `@input` of a button. Ideally of course this should not be an issue, but I doubt eslint has enough information to infer this.

If someone agrees, I could open a PR :)

### PR DESCRIPTION
**Pull request by rafaelss95**
…ria attributes

Currently the rule `accessibility-valid-aria` reports any word that **contain** the word "aria", like "variant", "arial", "ariaAnother", etc., while it should only report words that **start** with "aria-".

Fixes #271.

### CODE REVIEW

File: packages/eslint-plugin-template/tests/rules/accessibility-valid-aria.test.ts [hunk 1]
```diff
@@ -20,25 +20,26 @@ const messageId: MessageIds = 'accessibilityValidAria';
...
-        <input [aria-labelby]="text">
-               ~~~~~~~~~~~~~~~~~~~~~
+        <div aria-roledescriptio="text">Text</div>
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~
+        <input [aria-labelby]="label">
+               ^^^^^^^^^^^^^^^^^^^^^^               
+        <input [attr.aria-requiredIf]="required">
+               #################################
       `,
+      messages: [
```
Comment (rafaelss95) [inline packages/eslint-plugin-template/tests/rules/accessibility-valid-aria.test.ts:38]: @JamesHenry is there anyway to pass `data` when having multiple assertions?

**Comment by rafaelss95** [comment]
While I was fixing this I noticed that we could also expand this rule to report incorrect values, ex:

`<div aria-orientation="test"></div>`

This should be reported since the allowed values are `undefined`, `'horizontal'` and `'vertical'` as we could see in https://github.com/A11yance/aria-query/blob/e4b6d28df777c5e6f2a14fa3cdfba74bc3007e19/src/ariaPropsMap.js#L143-L-147.

WDYT, @JamesHenry? If so, I could send another PR for this.

File: packages/eslint-plugin-template/tests/rules/accessibility-valid-aria.test.ts (hunk 1 above)
Comment (JamesHenry) [reply packages/eslint-plugin-template/tests/rules/accessibility-valid-aria.test.ts:37]: I can't remember I'm afraid :)

**Comment by JamesHenry** [comment]
@rafaelss95 yeah that sounds like a good enhancement which still fits under the scope of the rule name!

**Pull request merged**